src/plugins/AvatarVCardPlugin.ts

```diff
@@ -60,6 +60,10 @@ export default class AvatarVCardPlugin extends AbstractPlugin {
             }
 
             let sha1OfAvatar = photo.text();
+            if (sha1OfAvatar!==null&&sha1OfAvatar.trim().length===0)
```

Comment:
What was the reason for this check? Could also be written as `let sha1OfAvatar = photo.text()?.trim() || null;`

Reply:
i want to be sure that sha1OfAvatar is null for "no photo given" if there are some whitespaces in the phototag